soc_086_subnational_hdi/README.md

```diff
@@ -0,0 +1,18 @@
+This file describes the data pre-processing that was done to [Subnational Human Development Index](https://globaldatalab.org/shdi/view/shdi/) for [display on Resource Watch](https://bit.ly/3tCh7JU).
+
+The source provided two files, a csv file containing the shdi data and a shapefile containing subnational boundaries used to calculate the index.
+
+The following steps describe the processing performed by the Resource Watch team to the dataset:
+
+1. The shapefile was read as a geopandas dataframe. The columns were formatted to comply with Carto's requirements.
```

Comment:
I feel like this was saying the same thing as the 4th step, but let me know if these should be more specific to describe separate steps.
```suggestion
1. The shapefile was read as a geopandas dataframe.
```